cpp/cppsdk/GSDK_CPP_Linux.vcxproj

```diff
@@ -50,6 +68,19 @@
       <LibraryDependencies>curl;jsoncpp;pthread</LibraryDependencies>
     </Link>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='DebugPIC|x64'">
+    <ClCompile>
+      <PreprocessorDefinitions>GSDK_LINUX</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)include;/usr/include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <CppLanguageStandard>c++1y</CppLanguageStandard>
+      <AdditionalOptions>-I $(RemoteProjectDir) -I $(RemoteProjectDir)/include %(AdditionalOptions)</AdditionalOptions>
+      <PositionIndependentCode>true</PositionIndependentCode>
```

Comment:
Quick question: Are we using PIC to get around some linker issues or is this for load time optimization ? Would be really helpful if you can provide some additional context on what the intended use case is for this config.

Reply:
Simply project structure requires that GSDK is linked into dynamic library. There is no additional reasons.

Reply:
Ic. And the current static lib configuration cannot be linked into the dynamic library ? Or are we trying to avoid the overhead of load time relocation with dynamic library ? Either ways - i see no issues with the change. Just trying to understand why the current configs cannot be linked into a dynamic library. :)

Reply:
Static library without PIC can not be linked into dynamic library. I do not know a way at least.

Reply:
Yep - did some light reading and could not find another way either. We're good here. Thanks !